pkg/template/instructions.go

```diff
@@ -58,7 +58,11 @@ func (is *InstructionSet) NewEndCtxNone() Instruction {
 
 func (is *InstructionSet) NewStartNodeAnnotation(nodeTag NodeTag, ann Annotation) Instruction {
 	collectedArgs := is.CollectNodeAnnotation.WithArgs(ann.Content).AsString()
-	return is.StartNodeAnnotation.WithArgs(nodeTag.AsString(), `"`+string(ann.Name)+`"`, ann.Position.AsIntString(), collectedArgs)
+	annLineNum := "None"
+	if ann.Position.IsKnown() {
+		annLineNum = ann.Position.AsIntString()
+	}
```

Comment:
Two suggestions in one:
- defer to Starlark itself for the name of `None`: it's _that_ package's responsibility. Also "documents" this dependency on a Starlark thing via an `import`
- is the "either-or" aspect (slightly) more readily clear with the `else`?

```suggestion
	var annLineNum string
	if ann.Position.IsKnown() {
		annLineNum = ann.Position.AsIntString()
	} else {
		annLineNum = starlark.None.String()
	}
```